api/src/user/entities/user.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+enum Gender {
+  male = 'male',
+  female = 'female',
+}
+
+@Entity('user')
+export class UserEntity {
```

Comment:
should be image specified

recommend for simple storage use string type (base64 encoding)